assets/styles/style.css

```diff
@@ -706,3 +725,153 @@ footer {
 }
 
 /* ====== CONTACT FORM END ==== */
+
+/* ====== CMODAL POPUP START ==== */
+.closeModal {
+  display: none;
+}
+
+.showModal {
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  padding: 2em 1em;
+  background: #97949371;
+  backdrop-filter: blur(14px);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.project-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: var(--white);
+  padding: 1.4em 1em;
+  color: var(--black);
+  width: 95%;
+  height: 100%;
+  overflow: scroll;
```

Comment:
- [ ] Nice work so far 👍🏿  just as the previous reviewer pointed out, your modal still needs a adjustment with margins and padding, as the button below are cut off from the popup

![image](https://github.com/Errytagedesign/microverse-portfolio/assets/68823331/a8b16616-85f0-4c62-abdb-2e907f49bb91)